schema/release.json

```diff
@@ -0,0 +1,55 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "node_modules/release-it/schema/release",
```

Comment:
Why does this value contain `node_modules`? Is it supposed to reference itself?

Reply:
it's just a unique id, we can write we want, i've just change it for a better in latest commit